### PR TITLE
update tabs (isWrapped, extend click area, Badge position)

### PR DESCRIPTION
**Pull request by Tuxosaurus**
No visual change but allows for simpler customisation when used.
Screen Shot 2019-07-22 at 18 38 13

### CODE REVIEW

File: src/tabs/index.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ interface TabsProps {
   readonly onChange?: Function
   readonly status?: TabStatus
   readonly className?: Classcat.Class
+  readonly isWrapped?: boolean
```
Comment (Tuxosaurus) [inline src/tabs/index.tsx]: I admit I'm not a big fan of my naming here. If you have other suggestions...

File: src/tabs/style.ts [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
 const bottomBorderWidth = '2px'
 const bottomBorderOffset = '1px'
-const badgeOffset = '2px'
 
 export default css`
+  .kirk-tabs-wrapper {
+    position: relative;
+  }
+
+  .kirk-tabs-wrapper::before {
```
Comment (wakooka) [inline src/tabs/style.ts]: Can't we use a normal border now that we this extra class? Or maybe we need an extra element again:
Main container could extend to 100%. And you could have a inner container with a max-width just for placing the tabs which would be the current `.wrapped` class

Comment (wakooka) [reply src/tabs/style.ts]: That way we make sure we don't break the layout if tabs need to be in a column for example

Comment (Tuxosaurus) [reply src/tabs/style.ts]: Alright, will try that way.

Comment (Tuxosaurus) [reply src/tabs/style.ts]: Done, also fixed the selected tab border issue in scrollable mode so now borders look the same as in fixed.
Screen Shot 2019-07-23 at 16 34 55

**Pull request merged**